src/main/resources/ink/abb/pogo/scraper/gui/index.html

```diff
@@ -338,10 +439,22 @@
     <div id="window-map" class="window">
         <span class="title">Map</span>
         <div class="content">
-            <div>
-                <input type="button" id="gpx_upload_button" value="Upload GPX file">
-                <input type="file" id="gpx_upload" accept=".gpx">
-                <input type="checkbox" id="map_autofollow" checked><label for="map_autofollow">Autofollow</label>
+            <div style="display: block;" align="center">
+                <div class="map-button">
+                    <div class="gpx-upload">
+                        <input type="file" id="gpx_upload">
+                    </div>
+                </div>
+                <div class="map-button">
+                    <h4>Autofollow bot</h4>
+                    <div style="display: block" class="autofollow-slide">
+                        <input type="checkbox" value="None" id="autofollow-slide" name="check" checked />
+                        <label for="autofollow-slide"></label>
+                    </div>
+                </div>
+                <div class="map-button">
+                    <div class="btn" id="pokevision-button">Pokévision</div>
```

Comment:
We're not sure, they said they are abiding by Niantic's wishes and hope that the rules will change in the future.  Check out their twitter twitter.com/PokeVisionGo